spring-integration-http/src/main/java/org/springframework/integration/http/outbound/HttpRequestExecutingMessageHandler.java

```diff
@@ -79,40 +52,12 @@
  * @author Gunnar Hillert
  * @author Artem Bilan
  * @author Wallace Wadge
+ * @author Shiliang Li
  * @since 2.0
  */
-public class HttpRequestExecutingMessageHandler extends AbstractReplyProducingMessageHandler {
-
-	private final Map<String, Expression> uriVariableExpressions = new HashMap<>();
-
+public class HttpRequestExecutingMessageHandler extends AbstractHttpRequestExecutingMessageHandler {
```

Comment:
Change to https://github.com/spring-projects/spring-integration/pull/2043 based on your suggestion.